packages/wix-ui-core/src/components/tooltip/Tooltip.tsx

```diff
@@ -53,7 +55,10 @@ export interface TooltipProps {
   minWidth?: number;
   /** content element maxWidth value */
   maxWidth?: number;
-
+  /** whether controlled or not*/
+  controlled?: boolean;
+  /** when controlled enables tooltip show or not*/
+  show?: boolean;
```

Comment:
```suggestion
  shown?: boolean;
```
let's keep consistency with `Popover`